examples/go/module-example/go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/fission/fission/examples/go/go-module-example
 
 require github.com/golang/example v0.0.0-20170904185048-46695d81d1fa
+
+go 1.13
```

Comment:
Should we move this change out of this PR?

Reply:
Sure, got this added into git staged my mistake, will fix in next commit.